### PR TITLE
feat(eigen-client-extra-feature): rlp encoding

### PR DESCRIPTION
**Pull request by juan518munoz**
## What ❔

Implement rlp encoding for the dispersed blob.

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `zkstack dev fmt` and `zkstack dev lint`.

### CODE REVIEW

**Review by gianbelinche — APPROVED** [review]
LGTM

File: core/node/da_clients/src/eigen/blob_info.rs [hunk 1]
```diff
@@ -0,0 +1,419 @@
...
+}
+
+impl Encodable for BlobInfo {
+    fn rlp_append(&self, s: &mut RlpStream) {
+        s.begin_list(2);
+        s.append(&self.blob_header);
+        s.append(&self.blob_verification_proof);
+    }
+}
+
```
Comment (juanbono) [inline core/node/da_clients/src/eigen/blob_info.rs:405]: Add tests for decoding/encoding

**Review by juanbono — CHANGES_REQUESTED** [review]
Looks great but add some unit tests for RLP encoding/decoding.

**Comment by juan518munoz** [comment]
> Looks great but add some unit tests for RLP encoding/decoding.

A simple test that encodes and decodes a blob has been added, do you think it's enough?

File: core/node/da_clients/src/eigen/blob_info.rs [hunk 2]
```diff
@@ -0,0 +1,505 @@
...
+            )?,
+        })
+    }
+}
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[tokio::test]
```
Comment (juanbono) [inline core/node/da_clients/src/eigen/blob_info.rs:425]: it doesn't need to be run async right?

Comment (juan518munoz) [reply core/node/da_clients/src/eigen/blob_info.rs]: It doesn't, removed it!

**Review by juanbono — APPROVED** [review]
LGTM

**Pull request merged**